README.adoc

```diff
@@ -206,84 +241,46 @@ gem 'asciidoctor'
-
-TIP: Your system may be configured to automatically update rpm packages, in which case no action is required by you to update the gem.
-
-==== apt-get (Debian or Ubuntu)
-
-To install the gem on Debian or Ubuntu, open a terminal and type:
+ $ sudo apk add -u asciidoctor
 
- $ sudo apt-get install -y asciidoctor
+=== APT
 
 To upgrade the gem, use:
 
  $ sudo apt-get upgrade -y asciidoctor
 
-TIP: Your system may be configured to automatically update deb packages, in which case no action is required by you to update the gem.
-
-The version of Asciidoctor installed by the package manager (apt-get) may not match the latest release of Asciidoctor.
-Consult the package repository for your distribution to find out which version is packaged per distribution release.
-
-* https://packages.debian.org/search?keywords=asciidoctor&searchon=names&exact=1&suite=all&section=all[asciidoctor package by Debian release]
-* https://packages.ubuntu.com/search?keywords=asciidoctor&searchon=names&exact=1&suite=all&section=all[asciidoctor package by Ubuntu release]
-
-[CAUTION]
-====
-You're advised against using the `gem update` command to update a gem managed by the package manager.
-Doing so puts the system into an inconsistent state as the package manager can no longer track the files (which get installed under /usr/local).
-Simply put, system gems should only be managed by the package manager.
-
-If you want to use a version of Asciidoctor that is newer than what is installed by the package manager, you should use https://rvm.io[RVM] to install Ruby in your home directory (i.e., user space).
-Then, you can safely use the `gem` command to install or update the Asciidoctor gem.
-When using RVM, gems are installed in a location isolated from the system.
-====
-
-==== apk (Alpine Linux)
-
-To install the gem on Alpine Linux, open a terminal and type:
-
- $ sudo apk add asciidoctor
+=== DNF
 
 To upgrade the gem, use:
 
- $ sudo apk add -u asciidoctor
-
-TIP: Your system may be configured to automatically update apk packages, in which case no action is required by you to update the gem.
-
-=== (d) Homebrew (macOS)
-
-You can use Homebrew to install Asciidoctor on macOS.
-If you haven't yet installed Homebrew, follow the instructions at https://brew.sh/[brew.sh].
-
-Once Homebrew is installed, you can install the `asciidoctor` gem using the http://brewformulas.org/Asciidoctor[asciidoctor] package.
-To do so, open a terminal and type:
+ $ sudo dnf update -y asciidoctor
 
- $ brew install asciidoctor
+=== Homebrew (macOS)
 
 To upgrade the gem, use:
 
  $ brew update
  $ brew upgrade asciidoctor
 
-TIP: Homebrew installs the `asciidoctor` gem into an exclusive prefix that's independent of system gems.
+=== gem update
+
+If you previously installed Asciidoctor using the `gem` command, you'll need to manually upgrade Asciidoctor when a new version is released.
+You can upgrade the gem by typing:
 
-=== Other installation options
+ $ gem update asciidoctor
```

Comment:
I want to drop this whole `gem update` thing since it doesn't actually work in practice. It does exactly the same thing as `gem install`. Let's change this to:

```
$ gem install asciidoctor && gem cleanup asciidoctor
```

That's the correct upgrade sequence.

Reply:
In other words, show this command but drop the whole explanation of cleanup as that's just not necessary to explain.